s3repo/view.py

```diff
@@ -27,42 +28,43 @@ def _readable_size(size):
                 return readable_size
             size /= 1024.0
 
-        return "%3.1f B" % (size)
+        return "%3.1f B" % size
 
     @staticmethod
-    def _get_directory(path, items):
+    def _get_directory(path: str, items: List) -> Response:
         """Display directory content as a HTML page."""
         displayed_path = path
         parent_path = '/'.join(path.split('/')[:-2])
 
         # Make size human readable.
-        readable_items = []
+        readable_items: List = []
         for item in items:
             if item.Size != '':
                 item = item._replace(Size=S3View._readable_size(item.Size))
             readable_items.append(item)
 
-        list_parameters = {'displayed_path': '/' + displayed_path,
-                           'path': path,
-                           'parent_path': parent_path,
-                           'items': readable_items}
+        list_parameters = {
+            'displayed_path': '/' + displayed_path,
+            'path': path,
+            'parent_path': parent_path,
+            'items': readable_items
+        }
 
         return render_template('index.html', **list_parameters)
 
     @staticmethod
-    def _get_file(path, response):
+    def _get_file(path, response) -> Response:
```

Comment:
path and response are not described

Reply:
path - `str`
response - described in https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3.html#S3.Client.get_object as I understand.